backend/controller/timeline/timeline_test.go

```diff
@@ -72,21 +75,60 @@ func TestTimelineDAL(t *testing.T) {
 		Response:      []byte(`{"time":"now"}`),
 		DestVerb:      schema.Ref{Module: "time", Name: "time"},
 	}
+
 	t.Run("InsertCallEvent", func(t *testing.T) {
-		err = timeline.insertCallEvent(ctx, callEvent)
+		call := callEventToCall(callEvent)
+		timeline.InsertCallEvent(ctx, call)
 		assert.NoError(t, err)
 	})
 
 	logEvent := &LogEvent{
-		Time:          time.Now().Round(time.Millisecond),
-		DeploymentKey: deploymentKey,
-		RequestKey:    optional.Some(requestKey),
-		Level:         int32(log.Warn),
-		Attributes:    map[string]string{"attr": "value"},
-		Message:       "A log entry",
+		Log: Log{
+			Time:          time.Now().Round(time.Millisecond),
+			DeploymentKey: deploymentKey,
+			RequestKey:    optional.Some(requestKey),
+			Level:         int32(log.Warn),
+			Attributes:    map[string]string{"attr": "value"},
+			Message:       "A log entry",
+		},
 	}
 	t.Run("InsertLogEntry", func(t *testing.T) {
-		err = timeline.InsertLogEvent(ctx, logEvent)
+		err = timeline.InsertLogEvent(ctx, &logEvent.Log)
+		assert.NoError(t, err)
+	})
+
+	ingressEvent := &IngressEvent{
+		DeploymentKey:  deploymentKey,
+		RequestKey:     optional.Some(requestKey),
+		Verb:           schema.Ref{},
+		Method:         "GET",
+		Path:           "/echo",
+		StatusCode:     200,
+		Time:           time.Now().Round(time.Millisecond),
+		Request:        []byte(`{}`),
+		RequestHeader:  []byte("{}"),
+		Response:       []byte(`{}`),
+		ResponseHeader: []byte("{}"),
```

Comment:
Two things:
1. Should we add another test with more content in these fields to make sure all the json marshaling works correctly?
2. I feel like I'm probably missing something but why do some of these have " and others have `?

Reply:
1. You bet!
2. Nah, just copy/pasta errors :) Thanks for flagging!

Reply:
This was surprisingly hard to decipher the json marshalling 😂

Reply:
hehehe